tests/store/test_sqlalchemy_store.py

```diff
@@ -307,7 +308,7 @@ def test_create_run(self):
         self.assertEqual(actual.info.start_time, expected["start_time"])
         self.assertEqual(len(actual.data.tags), 3)
 
-        name_tag = models.SqlTag(key='mlflow.runName', value='booyya').to_mlflow_entity()
+        name_tag = models.SqlTag(key=MLFLOW_RUN_NAME, value='booyya').to_mlflow_entity()
```

Comment:
This change is intended to be staged across two releases, but it may make sense to make more of the changes that 'assume source_type is in tags only'. These would include:

- Removing these parameters from the MlflowClient tracking API (e.g., mlflow/projects/__init__.py#create_run would have to embed these in tags instead).
- Mentioning in the protos that these fields are deprecated and moved to tags.
- Changing the JavaScript to read first from the tag and then from the field.

Reply:
We agreed that a subsequent PR should implement these.